### PR TITLE
relink the  persistent volume of petset

### PR DESCRIPTION
**Pull request by jianglingxia**
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.8 Features: set Milestone to `1.8` and Base Branch to `release-1.8`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5582)

### CODE REVIEW

**Comment by jianglingxia** [comment]
/assign @zacharysarah
PTAL thanks

**Comment by zacharysarah** [comment]
@jianglingxia :wave: Please resolve the merge conflicts in this branch. Thanks!

**Comment by jianglingxia** [comment]
@zacharysarah  I have rebased it,for your approve,thanks!

File: docs/concepts/workloads/controllers/petset.md [hunk 1]
```diff
@@ -24,7 +24,9 @@ Throughout this doc you will see a few terms that are sometimes used interchange
 
 * Node: A single virtual or physical machine in a Kubernetes cluster.
 * Cluster: A group of nodes in a single failure domain, unless mentioned otherwise.
-* Persistent Volume Claim (PVC): A request for storage, typically a [persistent volume](/docs/concepts/storage/persistent-volumes/walkthrough/).
+
+* Persistent Volume Claim (PVC): A request for storage, typically a [persistent volume](/docs/concepts/storage/persistent-volumes/).
+>>>>>>> relink the  persistent volume of petset
```
Comment (zacharysarah) [inline docs/concepts/workloads/controllers/petset.md]: Please delete this line.

Comment (jianglingxia) [reply docs/concepts/workloads/controllers/petset.md]: thanks,I have rebased it!

**Comment by xiangpengzhao** [comment]
It would be good to delete the note and image in template before submitting the pull request in the future :)

**Comment by jianglingxia** [comment]
OK，thanks for your advice@xiangpengzhao

**Pull request merged**